src/main/java/com/minelittlepony/bigpony/mod/CameraPresets.java

```diff
@@ -24,6 +26,13 @@ public float getDistance() {
     }
 
     public boolean isEqual(float height, float distance) {
-        return FloatUtils.equals(this.height, height) && FloatUtils.equals(this.distance, distance);
+        return FloatUtils.equals(this.height, height)
+                && FloatUtils.equals(this.distance, distance);
+    }
+
+    public boolean isEqual(float scaleX, float scaleY, float scaleZ) {
+        return this.height == scaleX
```

Comment:
Don't compare floating points with `eq`. Use the above method as an example of what to do.